config/migration/v1/schema.go

```diff
@@ -0,0 +1,41 @@
+package v1
+
+type Config struct {
+	Github    GithubConfig `json:"github"`
+	Poll      PollConfig   `json:"poll"`
+	Tracking  Tracking     `json:"tracking"`
+	ItemCount int          `json:"itemCount"`
+	Darkmode  bool         `json:"darkMode"`
+}
+
+type GithubConfig struct {
+	GraphQL string `json:"gqlAPI"`
```

Comment:
why chnage the name?

Reply:
That's the schema for v1. At the time that was the name used